white_label/brands/mabulle/js/locales/fr.json

```diff
@@ -5,5 +5,13 @@
       "buttonLogin": "J'ai déjà un Cloud Personnel Cozy",
       "title": "Ma Bulle"
     }
+  },
+  "modals": {
+    "IconChangedModal": {
+      "description": "L'icône de l'application «Ma Bulle» a été modifiée."
```

Comment:
« Ma Bulle »

Reply:
Fixed in https://github.com/cozy/cozy-flagship-app/compare/489df6ad7083f78a6dff324b1780b642819a262e..f9e185be78c78b3a9d7ff64f6a360b4ac56cca92